Assets/_Scripts/WizardAttackMeans.cs

```diff
@@ -38,7 +38,6 @@ public class WizardAttackMeans : MonoBehaviour {
 
 			Debug.Log ("[Spell] Attack using " + SpellDB.attackIDnames [(int)id]);
 
-
 			wizardAnimator.SetBool ("isCasting", false);
```

Comment:
Don't commit a change if there is no change.
Read carefully in Source Tree before you push